android/jni/AppHost.cpp

```diff
@@ -221,22 +221,33 @@ void AppHost::HandleTouchInputEvent(const Eegeo::Android::Input::TouchInputEvent
 	m_pInputProcessor->HandleInput(event);
 }
 
-void AppHost::Update(float dt)
+void AppHost::UpdateCardboardProfile(float cardboardProfile[])
+{
+    m_pApp->UpdateCardboardProfile(cardboardProfile);
+}
+
+void AppHost::MagnetTriggered(){
+    m_pApp->MagnetTriggered();
+}
+
+void AppHost::Update(float dt, float headTansform[])
 {
 	m_pInputProcessor->Update(dt);
-	m_pApp->Update(dt);
+	m_pApp->Update(dt, headTansform);
 }
 
-void AppHost::Draw(float dt)
+void AppHost::Draw(float dt, float headTansform[])
 {
-	m_pApp->Draw(dt);
+	m_pApp->Draw(dt, headTansform);
```

Comment:
Why is headTransform being passed to both Update() and Draw()? I think our usual pattern is update a camera during Update() and then pass its state into Draw(). Can we follow a similar pattern in VR, passing the headTransform into Update() only?